renderer.php

```diff
@@ -71,6 +71,165 @@ public function render_ratingallocate_strategyform($mform) {
+
+        $starturl = new moodle_url($this->page->url, array('action' => ACTION_START_DISTRIBUTION));
+        $deleteurl = new moodle_url($this->page->url, array('id' => $coursemoduleid, 'action' => ACTION_DELETE_ALL_RATINGS));
+
+        // Get description dependent on status.
+        $descriptionbaseid = 'modify_allocation_group_desc_';
+        $description = get_string($descriptionbaseid . $status, RATINGALLOCATE_MOD_NAME);
+
+        // Create start algorithm button.
+        $button = new single_button($starturl, get_string('start_distribution', RATINGALLOCATE_MOD_NAME), 'get');
+        // Enable only if the instance is ready and the algorithm may run manually.
+        $button->disabled = !($ratingover) || $isdistributionrunning;
+        $button->tooltip = get_string('start_distribution_explanation', RATINGALLOCATE_MOD_NAME);
+        $button->add_action(new confirm_action(get_string('confirm_start_distribution', RATINGALLOCATE_MOD_NAME)));
+
+        // Create delete all ratings button.
+        $deletebutton = new single_button($deleteurl, get_string('delete_all_ratings', RATINGALLOCATE_MOD_NAME, 'get'));
+        // Only allow deletion if new submission is possible and distribution currently not running.
+        $deletebutton->disabled = $ratingover || $isdistributionrunning;
+        $deletebutton->tooltip = get_string('delete_all_ratings_explanation', RATINGALLOCATE_MOD_NAME);
+        $deletebutton->add_action(new confirm_action(get_string('confirm_delete_all_ratings', RATINGALLOCATE_MOD_NAME)));
+
+        $table = new html_table();
+
+        // Add status, buttons for manual and algorithmic allocation and delete all ratings buuton to the table.
+        $this->add_table_row_triple($table,
+            $this->format_text($description),
+            $this->render($button) . '<br/>' . '<br/>' . $this->single_button(
+                new moodle_url(
+                    '/mod/ratingallocate/view.php',
+                    array('id' => $coursemoduleid,
+                    'action' => ACTION_MANUAL_ALLOCATION)),
+                    get_string('manual_allocation_form',
+                        RATINGALLOCATE_MOD_NAME),
+                    'get',
+                    array('disabled' => !$ratingover || $isdistributionrunning)
+                ),
+            $this->render($deletebutton)
+        );
+
+        if (has_capability('mod/ratingallocate:distribute_unallocated', context_module::instance($coursemoduleid))) {
+
+            if ($ratingover && $undistributeduserscount != 0 && !$isdistributionrunning) {
+
+                // Add empty row.
+                $this->add_table_row_triple($table, '', '', '');
+
+                $distributeunallocatedurleq = new moodle_url(
+                    $this->page->url,
+                    array('action' => ACTION_DISTRIBUTE_UNALLOCATED_EQUALLY)
+                );
+                $buttondisteq = new single_button($distributeunallocatedurleq,
```

Comment:
The buttons are very large for the German version. The string is longer but it seems the button size is larger than it has to be. Can you have a look at that?

Reply:
I think it is not possible to make the buttons smaller manually, since this is just how they look like in this theme with a word wrap. I could give them a fixed size, but that wouldn't look good on different sized windows of the page.